### ISSUE TITLE
Homebridge beta 1.3.3-beta.1 characteristic warning

### ISSUE THREAD

**Issue by grzegorz914** [homebridge/homebridge#2857]
Hi,

updated to 1.3.3-beta.1 and get this warning:

`[5.03.2021, 06:38:52] [homebridge-enphase-envoy] This plugin generated a warning from the characteristic 'Power': characteristic was supplied illegal value: number -0.013 exceeded minimum of 0. See https://git.io/JtMGR for more info.`

but I use custom characteristic and the min. value is not exceeded:

```
//Envoy production/consumption characteristics
  Characteristic.enphasePower = function () {
    Characteristic.call(this, 'Power', Characteristic.enphasePower.UUID);
    this.setProps({
      format: Characteristic.Formats.FLOAT,
      unit: 'kW',
      minValue: -1000,
      maxValue: 1000,
      minStep: 0.001,
      perms: [Characteristic.Perms.READ, Characteristic.Perms.NOTIFY]
    });
    this.value = this.getDefaultValue();
  };
  inherits(Characteristic.enphasePower, Characteristic);
  Characteristic.enphasePower.UUID = '00004111-000B-1000-8000-0026BB765291';
```

**Comment by oznu** [comment]
Thanks for raising this, I've fixed the bug and created a test case to prevent regression.

https://github.com/homebridge/HAP-NodeJS/commit/565ebd966455cea6cbaa426559136db6da6bef29#diff-141fa21699d2e874f90e7b50c5520f05879b5d657c4fc585142f3755be0a6ea1R977-R994

New beta build will be up shortly.

**Comment by oznu** [comment]
Fixed in  homebridge@1.3.3-beta.2 , thanks for catching this so quickly.
